src/components/Editor/Runners/PythonRunner/OutputViewToggle.scss

```diff
@@ -19,7 +20,10 @@
 
 .--light {
   .output-view-toggle {
-    background-color: $rpf-darkmode-grey95;
+    &__button {
+      background-color: $rpf-darkmode-grey95;
+    }
+
```

Comment:
One other tiny thing I've noticed is that this move of the background styling onto the button has changed the 'toggle-like' appearance, so now there is a gap between the corners of the buttons whereas before there wasn't 
i.e. before
<img width="91" alt="Screenshot 2023-02-22 at 12 58 44" src="https://user-images.githubusercontent.com/88904316/220626301-0167eeed-8e42-4bad-b014-643a83576bd2.png">
after
<img width="92" alt="Screenshot 2023-02-22 at 12 59 02" src="https://user-images.githubusercontent.com/88904316/220626373-0abfe2c7-13d0-4a83-85ea-bae24ceae718.png">

Reply:
Done in [3e4cf55](https://github.com/RaspberryPiFoundation/editor-ui/pull/373/commits/3e4cf559bd20b33449c1a512595a18fd1b3ad3d1) :) 

Ha yeah I always get surprised at what passes and fails colour contrast!